zerver/openapi/zulip.yaml

```diff
@@ -8984,6 +8984,15 @@ paths:
                           Whether the server needs upgrade.
 
                           **Changes**: New in Zulip 5.0 (feature level 74).
+                      event_queue_longpoll_timeout_seconds:
+                        type: integer
+                        description: |
+                          Present if `realm` is present in `fetch_event_types`.
+
+                          Client-side polling timeout for get_events, in seconds.
+
+                          **Changes**: New in Zulip 5.0 (feature level 74). Previously,
+                          this was called `poll_timeout` and only explicilty used by webapp.
```

Comment:
This is just new -- the web app `poll_timeout` field was not part of the API.  So we should instead say "Previously, this was hardcoded to 90 seconds, and clients should use that as a fallback value when interacting with servers where this field is not present."

Reply:
Yeah, I was a little confused on description to be used.